Writerside/topics/ES-Configuration.md

```
@@ -25,6 +25,9 @@
 | stats.enabled                          | Enables the stats module.                                                                                                                                                                                                                                                         | true or false                                                                                                                                                                                   | true                                                                 |
 | team.enabled                           | Enables the team module.                                                                                                                                                                                                                                                          | true or false                                                                                                                                                                                   | true                                                                 |
 | teleportation.enabled                  | Enables the teleportation module.                                                                                                                                                                                                                                                 | true or false                                                                                                                                                                                   | true                                                                 |
+| teleportation.back.wait                | The amount of time (in seconds) the player should have to wait for after sending the command before they're teleported. To disable this feature set this value to 0.	                                                                                                             | Integer                                                                                                                                                                                         | 3                                                                    |
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 208

(MD010, no-hard-tabs)